test/ipynb/mpl_results_graph.py

```diff
@@ -0,0 +1,200 @@
+# This code is part of Qiskit.
```

Comment:
I'm confused about why a whole new file was created for this? There is a lot of duplicated code here, I think it would be better to try integrate the graph results into the existing `mpl_results.py` file if possible

Reply:
@javabster I tried to actually use only one file `results.py` but because I am creating two files inside the `mpl` folder( `circuit` and `graph`) and the path to specific folders inside these have to be provided in order for the file to work properly, I was not able to integrate the whole code into one file. 
How about I remove the folders `graph` and `circuit` and store all the data in the same `.json` file and same `references` folder(like there won't be any different references and .py files for graph and circuit images). Then only one file will be able to do all the work.

Reply:
I think it can be worked into one file using the specific file names required, I will push a commit shortly to show you what I mean 😄

Reply:
Ok I've pushed some changes and it looks like it works now with just the original `results.py` so this file can be deleted

Reply:
Yeah, thanks I was kinda stuck there.